sal_interfaces/Dome/Dome_Commands.xml

```diff
@@ -1,241 +1,133 @@
-    <EFDB_Topic>Dome_command_stopAz</EFDB_Topic>
-    <Alias>stopAz</Alias>
-    <Device></Device>
-    <Property></Property>
-    <Action></Action>
-    <Value></Value>
-    <Explanation>http://sal.lsst.org</Explanation>
-</SALCommand>
-<SALCommand>
-    <Subsystem>Dome</Subsystem>
-    <Version>3.7.1</Version>
-    <Author></Author>
-    <EFDB_Topic>Dome_command_stopEl</EFDB_Topic>
-    <Alias>stopEl</Alias>
-    <Device></Device>
-    <Property></Property>
-    <Action></Action>
-    <Value></Value>
-    <Explanation>http://sal.lsst.org</Explanation>
-</SALCommand>
-<SALCommand>
-    <Subsystem>Dome</Subsystem>
-    <Version>3.7.1</Version>
-    <Author></Author>
-    <EFDB_Topic>Dome_command_stop</EFDB_Topic>
-    <Alias>stop</Alias>
-    <Device></Device>
-    <Property></Property>
-    <Action></Action>
-    <Value></Value>
-    <Explanation>http://sal.lsst.org</Explanation>
-</SALCommand>
-<SALCommand>
-    <Subsystem>Dome</Subsystem>
-    <Version>3.7.1</Version>
-    <Author></Author>
-    <EFDB_Topic>Dome_command_setTemperature</EFDB_Topic>
-    <Alias>setTemperature</Alias>
-    <Device></Device>
-    <Property></Property>
-    <Action></Action>
-    <Value></Value>
-    <Explanation>http://sal.lsst.org</Explanation>
-    <item>
-        <EFDB_Name>temperature</EFDB_Name>
-        <Description>Set the temperature to maintain in the dome heat sources</Description>
-        <IDL_Type>double</IDL_Type>
-        <Units>deg_C</Units>
-        <Count>1</Count>
-    </item>
-</SALCommand>
+<SALCommandSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALCommandSet.xsd">
+    <SALCommand>
+        <Subsystem>Dome</Subsystem>
+        <EFDB_Topic>Dome_command_crawlAz</EFDB_Topic>
+        <Description>Move the azimuth axis at constant velocity.</Description>
+        <item>
+            <EFDB_Name>velocity</EFDB_Name>
+            <Description>The velocity. Positive is the direction of increasing azimuth.</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units>deg/second</Units>
+            <Count>1</Count>
+        </item>
+    </SALCommand>
+    <SALCommand>
+        <Subsystem>Dome</Subsystem>
+        <EFDB_Topic>Dome_command_crawlEl</EFDB_Topic>
+        <Description>Move the elevation axis (light/wind screen) at constant velocity.</Description>
+        <item>
+            <EFDB_Name>velocity</EFDB_Name>
+            <Description>The velocity. Positive is the direction of increasing azimuth.</Description>
+            <IDL_Type>double</IDL_Type>
+            <Units>deg/second</Units>
+            <Count>1</Count>
+        </item>
+    </SALCommand>
+    <SALCommand>
+        <Subsystem>Dome</Subsystem>
+        <EFDB_Topic>Dome_command_moveAz</EFDB_Topic>
+        <Description>Move the azimuth axis at constant velocity, starting from a specified position.</Description>
+        <item>
+            <EFDB_Name>azimuth</EFDB_Name>

```

Comment:
Didn't we agree on slack to rename this to position?

Reply:
Great!